scripts/babel-relay-plugin/src/RelayQLAST.js

```diff
@@ -709,6 +709,37 @@ class RelayQLType {
     };
     return new RelayQLFragment(this.context, (generatedFragmentAST: any), this);
   }
+
+  getNodeIdField(): ?string {
+    const node = this.context.schema._typeMap['Node'];
+    if (node && node instanceof types.GraphQLInterfaceType) {
+      const fields = node._fields;
```

Comment:
Could you point me to where I should look for the correct API to access this GraphQL data, instead of `_typeMap` and `_fields`?